src/common/components/search-input/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+import styles from './search.css';
+
+export default function SearchInput(props) {
+  return (
+    <label htmlFor={ props.id }>
+      { props.label }
+      <input
+        id={ props.id }
+        className={ styles.selectRepositorySearch }
+        onChange={props.onChange}
```

Comment:
Just a nitpicking, but it would be nice to consistently use spaces inside of those values.

We don't really have a strict code style around it, so I'm fine with either `{props.onChange}` or `{ props.onChange }`, but it would be nice to have it consistently in whole component.